cypress/e2e/signIn.cy.js

```diff
@@ -2,10 +2,51 @@
 
 describe('Sign In page', () => {
   beforeEach(() => {
+    cy.visit('https://the-internet.herokuapp.com/login');
+  });
+
+  it('should be login with valid creds', () => {
+    cy.get('#username')
+      .type('tomsmith');
+
+    cy.get('#password')
+      .type('SuperSecretPassword!');
+
+    cy.get('.fa')
```

Comment:
it's hard to understand what are you clicking here, better to search buttons with such classes by text like
```suggestion
    cy.contains('.fa', 'sign in')
```